.scrutinizer.yml

```diff
@@ -5,7 +5,7 @@ tools:
     php_code_coverage: true
     external_code_coverage:
         timeout: 2400 # There can be another pull request in progress
-        runs: 3 # PHP 5.6 + PHP 7.0 + PHP 7.1
+        runs: 3 # PHP 5.6 + PHP 7.0 + PHP 7.1 + PHP 7.2
```

Comment:
Shouldn't this be 4? According to the comment and the count of the option it was 3 before adding PHP 7.2

Reply:
Yep, that's right!! Fixing it now